specification/graphrbac/data-plane/stable/1.6/graphrbac.json

```diff
@@ -2168,6 +2182,13 @@
           },
           "description": "The application permissions."
         },
+        "appRoles": {
+          "type": "array",
+          "items": {
+            "type": "string"
```

Comment:
please doublecheck. your other modification declare `"$ref": "#/definitions/appRole"` types as array elements but here you have type string. It is kind of inconsistent.